cas-server-support-jpa-ticket-registry/src/main/java/org/apereo/cas/config/JpaTicketRegistryConfiguration.java

```diff
@@ -47,6 +47,7 @@
     public String[] ticketPackagesToScan() {
         return new String[] {
                 "org.apereo.cas.ticket",
+                "org.apereo.cas.util.jpa",
```

Comment:
I am going to have to ask you, by the way, to be clearer in your future submissions plz. "Code refactoring introduced a small problem。" doesnt help anyone. What code refactoring? Which problem? How small? How does the solution address that problem?

Follow the PR template please.